docs/configuration_reference/index.rst

```diff
@@ -12,6 +12,7 @@ Configuration Parameters
     :maxdepth: 1
     :titlesonly:
 
+    behavior_version.rst
```

Comment:
So this should be above all others?

Reply:
Yes

Reply:
Hm I do not agree. This is definitely not the most important thing a newcomer would be interested in when learning about RETURNN (which is the purpose of the documentation).

Reply:
The configuration reference is not about learning RETURNN, but for looking up configuration parameters. And this will be one of the most important parameters, so I think it should be first. (e.g. when you write a new config, you need to think about setting this first before you do the rest, because other parameters might depend on this one).

Reply:
But I would rather say it's one of the least important parameters. In the remaining documentation, I would document the most recent behavior version always first, and if at all mention earlier behaviors when needed.

But also, I think this will be some feature which we would/should only rarely use. I expect most people will copy some example config or from someone else which just has this set to some recent version and otherwise will not care about it.

Also, I don't think people will need to look this up so often. Or if they need to, then this would be bad, because it should be mostly straight-forward. If it is not straight-forward, we did sth wrong.

But I don't really have a strong opinion on this. If you really think this should be first, then do it. But maybe someone else should also comment his/her opinion. @patrick-wilken @christophmluscher @Zettelkasten @jotix16 @Atticus1806 maybe? In the end, this is not really so important.

Reply:
While I don‘t use the section often, I would personally put it somewhere in the upper parts of the list. After version changes it might be searched for quite a bit.